CMakeLists.txt

```diff
@@ -61,7 +61,9 @@ option(static "Build static binaries." OFF)
 option(symbols "Build binaries in a way that allows easier debugging." OFF)
 option(warnings-as-errors "All warnings are treated as errors." ON)
 
-option(overlay "Build overlay." ON)
+if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
+	option(overlay "Build overlay." ON)
+endif()
```

Comment:
Instead of not creating this option on OpenBSD, we should simply set it's default to OFF there. In order to make it look pretty in the generated docs, use maybe e.g.
```cmake
set(is_openBSD ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
option(overlay "Build overlay." NOT ${is_openBSD})
```

Reply:
Sounds like a good idea.

The top-level `CMakeLists.txt` could set these `set_*` and we'd use them consistently rather than repeated string comparisons.

Reply:
> The top-level `CMakeLists.txt` could set these `set_*` and we'd use them consistently rather than repeated string comparisons.

In case we go this way, mind if I keep the commit in this diff as-is and then move everything to `is_*` in another PR?

Reply:
Yeah, leave it at this for now - as I wrote in your other PR: We probably should define some helper macros to deal with platform-specific options anyway :thinking: